test/CMakeLists.txt

```diff
@@ -327,6 +327,8 @@ if(CR)
     dynk_operators
     dynk_polynomial
     dynk_random
+    elensidealthck4d
+    elensidealthck6d
```

Comment:
rather than the thick tests, I would have added the ion one, but is fine also like this.

Reply:
I added those because I've seen them fail on nightly builds, that's all.